entity-framework/core/get-started/xamarin.md

```diff
@@ -187,6 +187,23 @@ using (var blogContext = new BloggingContext())
 }
 ```
 
+## iPhone Configuration
+
+Entity Framework Core sometimes generates dynamic code for performance reasons. This is not supported on the iOS platform and may cause the application to crash. The Xamarin team built an interpretor to handle these scenarios. It should be configured in your `.csproj` file in a `PropertyGroup`. Use a `Condition` to ensure the group is only valid for iPhone release builds:
+
+```xml
+<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|iPhone' ">
+</PropertyGroup>
+```
+
+Then add the `--interpreter` flag as an extra argument inside the group:
+
+```xml
+<CodesignExtraArgs>--interpreter</CodesignExtraArgs>
```

Comment:
Shouldn't it be `<MtouchExtraArgs>--interpreter</MtouchExtraArgs>`?
See my comment in #2666

Reply:
Good question. I'm confirming with @codemillmatt who has been working on the sample.

Reply:
Turns out the interpreter isn't the issue here after all - it is a linking issue. I'll update the code to make sure EF isn't linked out.